features/step_definitions/volunteer_op_steps.rb

```diff
@@ -22,4 +22,8 @@
   click_on 'Edit'
   fill_in('Description', :with => description)
   click_on 'Update a Volunteer Opportunity'
+end
+
+Given /^I should see (\d+) markers in the map$/ do |num|
+  expect(page).to have_css('.vol_op', count: num)
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.